comp/aggregator/demultiplexer/demultiplexerimpl/status_test.go

```diff
@@ -53,12 +55,17 @@ func TestStatusOutPut(t *testing.T) {
 		}},
 	}
 
+	mockTagger := taggerimpl.SetupFakeTagger(t)
```

Comment:
There is this, `noopimpl.NewTaggerClient()`, `nooptagger.Module()`, `fxutil.Test[tagger.Mock](b, taggerimpl.MockModule())`. Why for different ways to set up a tagger for tests?

Reply:
The tagger component mock is a bit of a mess. I would cleanup up on a following PR